thrift/src/main/thrift/atoms/chart.thrift

```diff
@@ -0,0 +1,83 @@
+namespace * contentatom.chart
+namespace java com.gu.contentatom.thrift.atom.chart
+
+enum ChartType {
+    BAR = 0
+    BARGROUP = 1
+    BARGROUPSTACK = 2
+    BARGROUPSTACK100 = 3
+    BAR100 = 4
+    BROKENBAR100 = 5
+    COL = 6
+    COLGROUP = 7
+    COLGROUPSTACK = 8
+    COLGROUPSTACK100 = 9
+    LINEDISCRETE = 10
+    LINECONTINUE = 11
+    PLOTDOT = 12
+    PLOTSCATTER = 13
+    ONBARDIFFARROW = 14
+    ONBARDIFFDOTS = 15
+    ONBARTICKS = 16
+    SLOPEGRAPH = 17
+}
+
+struct Range {
+    1: required double min
+    2: required double max
+}
+
+struct DisplaySettings {
+    1: required bool showHeadline = true
+    2: required bool showSource = true
+    3: optional bool showStandfirst
+    4: optional bool showLegend
+}
+
+struct Furniture {
+    1: required string headline
+    2: required string source
+    3: optional string standfirst
+}
+
+enum RowType {
+    STRING = 0,
+    DATE = 1
+}
+
+struct RowHeader {
+    1: required RowType rowType
+    2: required string value
+}
+
+struct Row {
+    1: required RowHeader rowHeader
```

Comment:
I would move this out and put it next to `columnHeaders`. That way we can eventually have more than one row of headers and more than one column of headers, e.g. 

Global header 1 | Global header 2
--------------- | -----------------
Sub 1 | Sub 2 | Sub 3 | Sub 4
----- | ------ | ------ | ------
1 | 2 | 3 | 4
5 | 6 | 7 | 8

Reply:
We're not sure how this would be represented in a graph? All the attempts we've had in Google Sheets gives us four columns, with no indication that the global headers even exists.

Do you have any examples of a graph that uses the global headers?